### DISCUSSION TITLE
My English language question keeps getting translated into Welsh

### DISCUSSION THREAD

**Discussion by crack-haddock** [openai/whisper#1201] [Q&A]
Sorry for asking this here, but I can't think of anywhere else to turn to.

In Open AI playground I have tried using speech to text to ask a question in my native (and only) tongue of English, but more often than not it is being translated into Welsh text (which I don't speak a word of). The translation appears perfect (according to Google translate) so there is definitely no problem in the comprehension of my voice and words.

And funnily enough, when I tried speaking the welsh for all the numbers 1 to 10, and a few phrases, it got translated into English gibberish!

I have seen hints that Whisper may be capable of detecting accents, and I do have a hint of a Welsh accent, but why the enforced translation? Obviously if I played around with the dev code, as you guys are doing, I think I could set a language default, but I can see no such setting in open ai playground.

Any ideas?

**Comment by pudepiedj** [comment]
There has been quite a bit of discussion of this phenomenon in the OpenAI Community Forum where I first reported it when an entire English audio was translated into some sort of Welsh; others then reported the same. Some contributors think it's solved, but I am not convinced. Take a look here:[https://community.openai.com/t/solved-whisper-translates-into-welsh/120780/79](url).
